CosNet.AppUI.Android/app/src/main/java/cosnet/android/ui/cosplayItem/ShowCosplayItem.java

```diff
@@ -58,15 +58,9 @@ protected void onCreate(Bundle savedInstanceState) {
     setWidgets();
     setListeners();
     togglelayout();
+    setListeners();
   }
 
-  private void setListeners() {
```

Comment:
waarom haal je de materials button weg?

Reply:
uhm? we didn't normally... kheb both geaccept